microprofile/src/test/java/io/quarkus/ts/openshift/microprofile/MicroProfileOpenShiftIT.java

```diff
@@ -21,20 +23,21 @@
 @AdditionalResources("classpath:jaeger-route.yaml")
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class MicroProfileOpenShiftIT extends AbstractMicroProfileTest {
+    @TestResource @WithName("jaeger-query")
+    private URL jaegerUrl;
+
     @TestResource
-    private OpenShiftClient oc;
+    private URL applicationUrl;
 
     @Test
     @Order(5) // see superclass
     public void verifyTracesInJaeger() {
-        String jaegerUrl = "http://" + oc.routes().withName("jaeger-query").get().getSpec().getHost();
-        String applicationUrl = "http://" + oc.routes().withName("microprofile-test").get().getSpec().getHost();
 
         // the tracer inside the application doesn't send traces to the Jaeger server immediately,
         // they are batched, so we need to wait a bit
         await().atMost(30, TimeUnit.SECONDS).untilAsserted(() -> {
             when()
-                    .get(jaegerUrl + "/api/traces?service=test-traced-service")
+                    .get(jaegerUrl.toExternalForm() + "/api/traces?service=test-traced-service")
```

Comment:
I think you can just rely on `URL.toString` calling `toExternalForm` (that's actually specified).

Reply:
I wanted to be specific, but can adjust for sure